osgar/drivers/rosproxy.py

```diff
@@ -65,6 +65,7 @@ def run(self):
 
 class ROSProxy(Thread):
     def __init__(self, config, bus):
+        bus.register('cmd_vel', 'imu_data', 'imu_data_addr')
```

Comment:
I suppose that now all drivers and modules have to call `bus.register()` ... this is relatively big change and will break old code, so I would add it into `CHANGELOG` (which I no longer remember how to generate). Maybe we should create release BEFORE this merge??

Reply:
(the original comment was supposed to be that I do not see `ROSMsg` module, but it was maybe note merged before this PR??)

Reply:
It has been merged 3 days ago and this is 7 days old.

Reply:
I am not sure which old code you are referring to. I have adjusted everything there was in master. I don't feel strongly either way about releases, I am afraid nobody uses them anyway.

Reply:
As far as I know at least two projects at the university use OSGAR ... but probably from git. It is probably not critical yet.